stories/Tooltip.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/no-children-prop */
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, select } from '@storybook/addon-knobs';
+import { Tooltip } from '@textkernel/oneui';
+import { POPUP_PLACEMENTS } from '@textkernel/oneui/constants';
+
+storiesOf('Molecules|Tooltip', module)
+    .addDecorator(withKnobs)
+    .add('Tooltip', () => (
+        <div>
+            <div style={{ display: 'table', margin: '0 auto' }}>
```

Comment:
why do we need these divs, and `display: table`? It is unlikely that we will ever use it in this way in a real application. Can we demonstrate the function by e.g. rendering it within `Text`?

Reply:
I added this style to move the component to the middle of the screen in order to all of the placement options be viable

Reply:
`margin: 0 auto` makes sense to me. but not `display: table`. and also don't see why we need 2 divs instead of just one.